QSB/QSBNetworkManager.cs

```diff
@@ -90,6 +96,24 @@ private string GetPlayerName()
 			}
 		}
 
+		/// create a new network prefab from the network object prefab template.
+		/// this works by calling Unload(false) and then reloading the AssetBundle,
+		/// which makes LoadAsset give you a new resource.
+		/// see https://docs.unity3d.com/Manual/AssetBundles-Native.html.
+		private static GameObject MakeNewNetworkObject(int assetId, string name, Type transformSyncType)
+		{
+			QSBCore.NetworkAssetBundle.Unload(false);
```

Comment:
why can't you just do Instantiate on the object loaded from the assetbundle?

Reply:
because im making prefabs, not objects. technically you could do that, but i like this way better because then the objects wont be in the scene, and they also be awakened until theyre actually instantiated